src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -138,13 +137,14 @@ class MediaChromeSelectMenu extends window.HTMLElement {
     const buttonRect = this.#button.getBoundingClientRect();
 
     // if we're outside of the controller,
-    // one of the components should have a media-controller attribute
-    // or the button wasn't slotted
+    // one of the components should have a media-controller attribute.
+    // There isn't a good way now to differentiate between default buttons or
+    // or a slotted button but outside of the media-controller.
+    // So, a regular declarative selectmenu may default to open up rather than down.
     if (
       this.hasAttribute('media-controller') ||
       this.#button.hasAttribute('media-controller') ||
-      this.#listbox.hasAttribute('media-controller') ||
-      !this.#buttonSlotted
+      this.#listbox.hasAttribute('media-controller')
```

Comment:
This example demonstrates it better: https://media-chrome-git-fork-gkatsev-no-light-dom-generated-els-mux.vercel.app/examples/control-elements/media-chrome-selectmenu.html
See buttons at the top. This makes it so that the menu opens down rather than up.
This top CC button is outside the media-controller and by default will open down.
![image](https://user-images.githubusercontent.com/535884/226771958-92b4b30f-c0e3-4575-b304-1e6c7acb3daa.png)